src/pythainer/builders/utils.py

```diff
@@ -243,3 +243,66 @@ def project_git_cmake_build_install(
         install=install,
         cleanup=cleanup,
     )
+
+
+def install_package_from_deb(
+    builder: PartialDockerBuilder,
```

Comment:
this parameter appears unused now. Also please respect the format of the docstring with the documentation of each parameter